### PR TITLE
Add support for `Window::theme` on the web

### PR DESCRIPTION
**Pull request by tronical**
- [ ] Tested on all platforms changed
- [x] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [x] Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior
- [ ] Created or updated an example program if it would help users understand this functionality
- [ ] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

### CODE REVIEW

**Comment by ogoffart** [comment]
I think this change is useful as is, so IMHO shouldn't prevent to be merged.  But it could also emit the ThemeChanged event when the media query changes (using for example `MediaQueryList::set_onchange`)

**Comment by tronical** [comment]
> I think this change is useful as is, so IMHO shouldn't prevent to be merged. But it could also emit the ThemeChanged event when the media query changes (using for example `MediaQueryList::set_onchange`)

Winit already implements the part of sending `ThemeChanged` when targeting the web. It's just missing the getter for the current value.

File: CHANGELOG.md
Comment (madsmtm) [inline CHANGELOG.md:13]: Nit: Write the platform first

```suggestion
- Web: Added support for `Window::theme`.
```

**Review by madsmtm — APPROVED** [review]
Only a minor nit with the changelog, otherwise looks fine

**Comment by tronical** [comment]
Good point, applied:). Feel free to choose merge with squash. (Or I can also force push a squash if you prefer)

**Comment by madsmtm** [comment]
Don't have a preference for how you do the commits, they'll be squashed once merged anyhow ;)

I'll need you to fix the conflicting changelog though, since I don't have write access to your fork.

**Comment by tronical** [comment]
Oops, not sure why that permission isn't there. I rebased, squashed, and resolved the conflict.

**Comment by madsmtm** [comment]
Thanks!

**Pull request merged**
